getting-started/forge-multi-node.md

```diff
@@ -0,0 +1,256 @@
+```
+
+To list added data nodes we can run `\des+` command if using `psql`. 
+
+```bash
+tsdb=> \des+
+List of foreign servers
+-[ RECORD 1 ]--------+------------------------------------------------------------------------------------
+Name                 | dn1
+Owner                | multinode
+Foreign-data wrapper | timescaledb_fdw
+Access privileges    | 
+Type                 | 
+Version              | 
+FDW options          | (host 'fd71nenmk8.c8mhe44nad.dev.metronome-cloud.com', port '34612', dbname 'tsdb')
+Description          | 
+```
+
+### Step 3b: Add a User Mapping for each data node
+
+Now we can create a `USER MAPPING `that will enable communication between the 
+access node and data node.
+
+```SQL
+CREATE USER MAPPING FOR tsdbadmin SERVER dn1 OPTIONS (user 'tsdbadmin', password 'tsdbadmin_user_password_for_DN1');
+```
+
+Repeat these steps for each additional data node that you want to add to the 
+cluster. **Always invoke these commands from the access node!**
+
+## Step 4: Create a distributed hyptertable
+
+Finally, can create a distributed hypertable and add data to verify everything is
+set up and working correctly.
+
+```SQL
+-- Create your regular table
+CREATE TABLE sensor_data (
+  time TIMESTAMPTZ NOT NULL,
+  sensor_id INTEGER,
+  temperature DOUBLE PRECISION,
+  cpu DOUBLE PRECISION
+);
+
+-- Convert the table to a distributed hypertable
+SELECT create_distributed_hypertable('sensor_data', 'time', 'sensor_id');
+
+-- Insert some test data to verify you are error free
+INSERT INTO sensor_data VALUES  ('2020-12-09',1,32.2,0.45);
+```
+
+One major point to recognize about the SQL above is the declaration of a partition 
+column (`sensor_id`) for the distributed hypertable. This is intentional, and 
+**recommended**, for distributed hypertable setups. Previously, with regular, 
+single-node hypertables, there was often little benefit in specifying a partition 
+key when creating the hypertable. With distributed hyptertables, however, adding
+a partition key is essential to ensure that data is distribute across data nodes 
+on something more than just time. Otherwise, all data for a specific time range 
+will go to one chunk on one node, rather than being distributed across each data 
+node for the same time range.
+
+### Adding another user (optional)
+
+One of the other advantages of user mapping based approach allows us to add 
+additional users to the multi-node cluster.
+
+From the **access node**, create a new user and GRANT all privileges. 
+
+```SQL
+CREATE ROLE mn_user1 LOGIN PASSWORD 'password';
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mn_user1;
+```
+
+While connected to the access node we can use a specially designed function to 
+execute SQL commands against data nodes:
+
+```SQL
+CALL distributed_exec(query => 'CREATE ROLE mn_user1 LOGIN PASSWORD $$password$$', node_list => '{dn1}');
```

Comment:
Maybe extend `node_list` to `{dn1,dn2}` and mention in the sentence above that we assume that you've previously added `dn1` and `dn2` as data nodes.

Reply:
Same  list would go for the query below of course